src/framework/mlt_property.h

```diff
@@ -31,7 +31,7 @@
 #endif
 
 #if defined(__GLIBC__) || defined(__APPLE__) || (__FreeBSD_version >= 900506)
```

Comment:
This #if needs to be broken up if you are addressing a glibc change. Also, I think there should be a glibc version check as well with perhaps older versions still #include xlocale.h.

Reply:
#if defined(__GLIBC__) || defined(__APPLE__) || (__FreeBSD_version >= 900506)
+#if __GLIBC_MINOR__ >= 26
+#include <locale.h>
+#else
 #include <xlocale.h>
+#endif
 #else
 typedef char* locale_t;
 #endif
is this ok?